backend/app/exceptions.py

```diff
@@ -338,3 +338,11 @@ class IndividualLeverageNotImplementedError(OctantException):
 
     def __init__(self):
         super().__init__(self.description, self.code)
+
+
+class InvalidDelegationForLockingAddress(OctantException):
+    code = 400
+    description = "Secondary address cannot lock any GLMs"
```

Comment:
In AC it states:

![image](https://github.com/user-attachments/assets/9e53f910-4bb6-492c-a1ec-95599b058062)

But let's leave it as is, not a huge difference.